Client/src/components/common/Sidebar/Linklst/index.css

```diff
@@ -0,0 +1,8 @@
+.li-links{
+  padding: 7px;
+}
+
+.link{
+  list-style-type: none;
+  padding-left: 10px;
+}
```

Comment:
add a new line at the end 
delete line 4 white space

Reply:
Done :+1: